src/udev/udev-event.c

```diff
@@ -6,6 +6,7 @@
 #include <net/if.h>
 #include <stddef.h>
 #include <stdlib.h>
+#include <sys/sdt.h>
```

Comment:
hmm, which package provides this header? i don't have it here?

Reply:
needs some ifdeffery i guess

Reply:
Good point, these needs to be ifdefed. Btw, it is systemtap-devel.